docs/xml.rst

```diff
@@ -0,0 +1,50 @@
+Polarion xml importer
+===================
+
+Usage
+--------------
+
+This chapter details operations to import data.
+
+
+Import an xml file of results
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Import an xml file of results (see xml_junit.xsd for the standard format) into a new or an existing test run in a project
+
+
+.. code:: python
+
+    import os
+    from polarion.xml import Config, Importer
```

Comment:
Add ResultExporter import here

Reply:
@jesper-raemaekers, anything is missing ?